packages/components/src/Components/Maintenance/icon-maintenance.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import './icon-maintenance.scss';
+const IconMaintenance = () => (
+    // eslint-disable-next-line max-len
+    <svg id="Icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 36 36" className='ins-i-maintenance'><title>Icon-Red_Hat-Hourglass-A-Red-RGB</title><path className="cls-1" d="M21,15.56A2.43,2.43,0,0,0,22.25,13v-.26h-8.5V13A2.43,2.43,0,0,0,15,15.56l3,1.59Z"/><rect className="cls-1" x="13.75" y="8.75" width="8.5" height="2.5"/><path className="cls-1" d="M22.25,23A2.43,2.43,0,0,0,21,20.44l-3-1.59-3,1.59A2.43,2.43,0,0,0,13.75,23v4.26h8.5Zm-3.56-.7a.78.78,0,0,1-.16.24.63.63,0,0,1-.24.16.74.74,0,0,1-.82-.16.63.63,0,0,1-.16-.24.72.72,0,0,1-.06-.29.75.75,0,0,1,1-.69.78.78,0,0,1,.24.16.75.75,0,0,1,.22.53A.72.72,0,0,1,18.69,22.29Z"/><path className="cls-2" d="M25,27.25H23.75V23a3.84,3.84,0,0,0-1.89-3.78l-.1-.07L19.6,18l2.16-1.15a.21.21,0,0,0,.1-.06A3.84,3.84,0,0,0,23.75,13V8.75H25a.75.75,0,0,0,0-1.5H11a.75.75,0,0,0,0,1.5h1.25V13a3.84,3.84,0,0,0,1.89,3.78.21.21,0,0,0,.1.06L16.4,18l-2.16,1.14-.1.07A3.84,3.84,0,0,0,12.25,23v4.26H11a.75.75,0,0,0,0,1.5H25a.75.75,0,0,0,0-1.5ZM22.25,8.75v2.5h-8.5V8.75ZM15,15.56A2.43,2.43,0,0,1,13.75,13v-.26h8.5V13A2.43,2.43,0,0,1,21,15.56l-3,1.59ZM13.75,23A2.43,2.43,0,0,1,15,20.44l3-1.59,3,1.59A2.43,2.43,0,0,1,22.25,23v4.26h-8.5Z"/><path className="cls-2" d="M18.29,21.31a.75.75,0,0,0-1,.69.72.72,0,0,0,.06.29.63.63,0,0,0,.16.24.74.74,0,0,0,.82.16.63.63,0,0,0,.24-.16.78.78,0,0,0,.16-.24.72.72,0,0,0,.06-.29.75.75,0,0,0-.22-.53A.78.78,0,0,0,18.29,21.31Z"/></svg>
```

Comment:
Can't we use PF `HourglassIcon` instead of this?

Reply:
@karelhala can you link me which one you're talking about? I can't find it in the PF icon page

Reply:
@mmenestr PF has really bad icon documentation, I've asked multiple times to list the full icon set if designers can't find any icon, but looks like it's still not there. You can find the full list here http://patternfly-react.surge.sh/icons. If you look for `Hourglass Half` you should see it (there's so much icons that the page sometimes breaks, it's basically this icon https://fontawesome.com/icons/hourglass-half?style=solid from font awesome), it doesn't look 100 % like the one used in here, but I'd prefer using these icons over custom ones as we don't have to maintain them and we can't really share the custom ones.

Reply:
![Screen Shot 2020-07-21 at 8 35 28 AM](https://user-images.githubusercontent.com/12520842/88055543-2bfa1d00-cb2d-11ea-935f-50502c9db9f0.png)

Reply:
Ok, after talking to some PF people, the recommendation is to stick to the PF icons since it's available in there! So if you could switch out the image to use this icon: that would be great (sorry Ryan).
Basically, I was looking at [this icons page](https://www.patternfly.org/v4/design-guidelines/styles/icons) which doesn't show every single icon available in font awesome, and that's why I didn't think there was an hourglass icon, and used the branding one.

Reply:
@mmenestr to be honest, that's exactly what I wanted to avoid by telling PF folks to pul link for a full list of icons. If you don't know that the page exist you can't know about these icons, simple as that...

Reply:
Updated! Reflected in top screenshot.